### PR TITLE
add link to release notes in nuspec

### PR DESCRIPTION
**Pull request by adamralph**
To simplify the forthcoming 2.0.0 release.

Is also a small step towards #665.

Connects to #336.

### CODE REVIEW

File: rakefile.rb [hunk 1]
```diff
@@ -47,8 +47,7 @@
     - [ ] attach nupkg
     - [ ] publish the release
 - [ ] push NuGet package
-- [ ] copy release notes from GitHub to NuGet
-- [ ] de-list pre-release or superseded buggy NuGet packages if present (copy any release notes forward to the new version)
```
Comment (blairconrad) [inline rakefile.rb]: Do we want, then, to leave all x.y.z-beta123 releases on GitHub and rely on them providing the release notes for the production release?
I'd think that we'd want to still consolidate all the notes onto the final release build, as we've done for 2.0.0.

**Comment by blairconrad** [comment]
Generally good, @adamralph. One small concern noted.

File: rakefile.rb (hunk 1 above)
Comment (adamralph) [inline rakefile.rb]: Agreed, I think we should continue to do that, but this time that consolidation will be in GitHub and not NuGet. I'll amend.

Comment (adamralph) [inline rakefile.rb]: @blairconrad I've amended so that we move the release notes forward in GitHub. How does it look now?

Comment (blairconrad) [inline rakefile.rb]: Like a million bucks. Do you want to squash, or are you happy with 3 commits?

Comment (adamralph) [inline rakefile.rb]: I'm happy with the 3 commits. They each have a distinct purpose and that's how I originally formed the PR. I just amended the last one based on this discussion.

Comment (adamralph) [inline rakefile.rb]: Arguably the first and third commits ought to be squashed to make the changes consistent (the second commit is an independent refactoring). I can do that if you want...

Comment (blairconrad) [inline rakefile.rb]: Nah, I'm not fussy. Just wanted you to have the opportunity, if you were really keen.

**Comment by blairconrad** [comment]
Thanks, @adamralph! Soon there'll be nothing for me to do on #665.
Seriously, though, I will do something.

File: rakefile.rb (hunk 1 above)
Comment (adamralph) [inline rakefile.rb]: :scream: 

image

I love it when a (micro-commit) plan comes together... :wink: 

I think it's mainly PR merges though. https://github.com/FakeItEasy/FakeItEasy/graphs/contributors doesn't take those into account.

Comment (blairconrad) [inline rakefile.rb]: I was gonna say that I didn't think we were that close. But merges are important too!

Comment (thomaslevesque) [inline rakefile.rb]: You can exclude the merge commits:

```
D:\Docs\GitHub\FakeItEasy [master ≡]> git shortlog -nse --no-merges
   447  Patrik Hägne <patrik@hagne.se>
   313  Blair Conrad <blair@blairconrad.com>
   219  Adam Ralph <adam@adamralph.com>
    41  Thomas Levesque <thomaslevesque@users.noreply.github.com>
    10  Johan Nilsson <jonh04@gmail.com>
    10  Philipp Dolder <philipp.dolder@gmail.com>
     9  crutt <ruttencutterdev@gmail.com>
     9  danielmarbach <daniel.marbach@openplace.net>
     8  Ulrich Buchgraber <ulrich.b@gmx.at>
     5  chillitom <tom.rathbone@gmail.com>
     4  Jimmy Headdon <jimmy.headdon@gmail.com>
     4  galaktor <galaktor@gmx.de>
     3  Nick Nicholls <nick.nicholls@citrix.com>
     2  Jacek Szumigaj <jacek@zwarszawy.pl>
     2  Jamie Humphries <j@miehumphri.es>
     2  Tim Lovell-Smith <tilovell@>
     2  Travis Illig <travis.illig@silverlining.ventures>
     1  Christian Merat <christian.merat@gsoft.com>
     1  Daniel West <daniel.j.west@gmail.com>
     1  Hurons <bjorn.ramakers@gmail.com>
     1  Igal Tabachnik <hmemcpy@gmail.com>
     1  Jamie Humphries <jamie.humphries@softwire.com>
     1  Jason Finch <jason.finch@gmail.com>
     1  Richard Bennett <richard@adaptpos.com>
     1  Tom Burton <tom@tburton.co.uk>
     1  Urs Enzler <urs.enzler@gmail.com>
D:\Docs\GitHub\FakeItEasy [master ≡]>
```

Comment (adamralph) [inline rakefile.rb]: Interesting. But that's also way different to https://github.com/FakeItEasy/FakeItEasy/graphs/contributors as well (see Patrik), so I don't really know where the latter gets its numbers...

**Pull request merged**